src/test/java/stepdefinitions/Hook.java

```diff
@@ -18,11 +18,11 @@ public class Hook {
 //        //driver.get(ConfigReader.readProperty("smart_bear_url"));
 //    }
 
-    @After
-    public void tearDown(Scenario scenario){
-
-        driver= DriverHelper.getDriver();
-        BrowserUtils.getScreenShotWithCucumber(driver, scenario);
-        driver.quit();
-    }
+//    @After
+//    public void tearDown(Scenario scenario){
+//
+//        driver= DriverHelper.getDriver();
+//        BrowserUtils.getScreenShotWithCucumber(driver, scenario);
+//        driver.quit();
+//    }
```

Comment:
I think you commented out the teardown. it will be better to keep it in there.

Reply:
It's not working with api steps but it's active now